app/components/sign-up-form/template.hbs

```diff
@@ -34,18 +34,14 @@
         aria-label=(t 'sign_up_form.password')
     }}
     <div class="row" hidden={{not progress}}>
-        <div class="col-xs-8">
+        <div class="col-xs-12">
```

Comment:
This is supposed to be 8 to be consistent with the current home page

Reply:
Please check the flowdock link in the PR description, but tl;dr it's changing on the core osf, so I figured I'd do that here.